### PR TITLE
Supervisor role

### PR DESCRIPTION
**Pull request by rhnonose**
- [x] Add supervisor as a role
- [x] Grant read-only permission to everything
- [x] Increment tests

### CODE REVIEW

File: lib/course_planner_web/templates/layout/supervisor_app_navbar.html.eex [hunk 1]
```diff
@@ -0,0 +1,23 @@
+<%= CoursePlannerWeb.SharedView.navbar "Course Planner" do %>
```
Comment (ghatighorias) [inline lib/course_planner_web/templates/layout/supervisor_app_navbar.html.eex]: is there a difference between coordinator and supervisor's navbar ?

**Comment by ghatighorias** [comment]
Just a point. i am not sure if it's needed for supervisor to get notification as well. cause it's rather looking at things. what do you think ?

**Comment by rhnonose** [comment]
@ghatighorias Since there's the option to have notification in x period of days, maybe we can add an option to disable it (or usa 0).

File: lib/course_planner_web/templates/layout/supervisor_app_navbar.html.eex (hunk 1 above)
Comment (rhnonose) [reply lib/course_planner_web/templates/layout/supervisor_app_navbar.html.eex]: Nope, though I might need to customize it so made a copy.

**Pull request merged**